static/workspace.html

```diff
@@ -197,14 +197,15 @@ <h2 class="columnHeader">Output:</h2>
       </div>
       <div id="buttonColumn" class="col-1 bg-white">
         <p class="columnContent">
-          <button id="show-button" data-toggle="modal" onclick="Prism.highlightAll()" href="#jsModal">Show JavaScript</button>
-          <button id="show-dashboard-button" data-toggle="modal" href="#jsModalRun">Open Dashboard</button>
-          <button id="run-button" onclick="runCode()">Run JavaScript</button>
+          <button type="button" class="btn btn-success btn-block" id="run-button" onclick="runCode()">Run JavaScript</button>
+          <button type="button" class="btn btn-danger btn-block" id="kill-button" onclick="terminateWorker()">Kill execution</button>
+          <button type="button" class="btn btn-warning btn-block" id="clear-button" onclick="clearOutput()">Clear Output</button>
+          <button type="button" class="btn btn-outline-dark btn-block" id="show-button" data-toggle="modal" onclick="Prism.highlightAll()" href="#jsModal">Show JavaScript</button>
+          <button type="button" class="btn btn-outline-dark btn-block" id="show-dashboard-button" data-toggle="modal" href="#jsModalRun">Dashboard</button>
           <!-- <button id="stop-button" onclick="stopWorker()">Stop execution</button> -->
-          <button id="kill-button" onclick="terminateWorker()">Kill execution</button>
           <!-- <button id="step-button" onclick="stepCode()">Step JavaScript</button> -->
           <!-- <button id="reset-button" onclick="generateCodeAndLoadIntoInterpreter()">Reset execution</button> -->
-          <button id="clear-button" onclick="clearOutput()">Clear Output</button>
+          
```

Comment:
Do you want to align the design of the commented buttons with the uncommented one, maybe? Just in case we're going to use it ;)